src/engine/Engine.cpp

```diff
@@ -1125,6 +1125,8 @@ void Engine::extractSolution( InputQuery &inputQuery )
             if ( _preprocessor.variableIsFixed( variable ) )
             {
                 inputQuery.setSolutionValue( i, _preprocessor.getFixedValue( variable ) );
+                inputQuery.setLowerBound( i, _preprocessor.getFixedValue( variable ) );
```

Comment:
Why are these calls added now? Are they related to abs constraint?

Reply:
These are small issues that were discovered as part of the testing of the Abs Constraint